core/app/serializers/uffizzi_core/api/cli/v1/projects/deployments_serializer.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class UffizziCore::Api::Cli::V1::Projects::DeploymentsSerializer < UffizziCore::BaseSerializer
+  type :deployment
+
+  attributes :id,
+             :created_at,
+             :updated_at,
+             :state,
+             :preview_url,
+             :metadata
+
+  belongs_to :deployed_by
+
+  def deployed_by
+    object.deployed_by
```

Comment:
Will this cause the N+1 problem?

Reply:
Honestly, I'm not sure. I just kept the style from  `deployment_serializer`, but as I tested right now this method can be safely removed(so I did this)
probably it can be removed from `deployment_serializer` too

Reply:
Let's remove then